pkg/controller/operator.go

```diff
@@ -2,21 +2,24 @@ package controller
 
 import (
 	"context"
+	"crypto/x509"
 	"encoding/json"
 	"fmt"
 	"os"
 
-	"k8s.io/client-go/rest"
-
 	"k8s.io/klog"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/pkg/version"
+	"k8s.io/client-go/rest"
 
 	configv1client "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
 	"github.com/openshift/library-go/pkg/controller/controllercmd"
 
+	v1 "github.com/openshift/api/config/v1"
```

Comment:
Mind using more descriptive name, ideally using `config` in it. `v1` is too vague IMO

Reply:
fixed